pkg/emptydisk/emptydisk_test.go

```diff
@@ -50,8 +33,10 @@ var _ = Describe("EmptyDisk", func() {
 
 	Describe("a vmi with emptyDisks attached", func() {
 		It("should get a new qcow2 image if not already present", func() {
-			vmi := api.NewMinimalVMI("testvmi")
-			AppendEmptyDisk(vmi, "testdisk")
+			vmi := libvmi.New(
+				libvmi.WithEmptyDisk("testdisk", v1.DiskBusUSB, resource.MustParse("3Gi")),
```

Comment:
This is not the same as before, it should be `libvmi.WithEmptyDisk("testdisk", "", resource.MustParse("3Gi")),`